app/uk/gov/hmrc/awrslookup/controllers/LookupController.scala

```diff
@@ -91,6 +90,7 @@ class LookupController @Inject()(val environment: Environment) extends BaseContr
         BadRequest(lookupResponse.body)
       case INTERNAL_SERVER_ERROR =>
         DoAuditing(apiType, "Search Result", "INTERNAL_SERVER_ERROR", eventTypeInternalServerError, metrics.incrementFailedCounter)
+        Logger.error(lookupResponse.body)
```

Comment:
Please add a reference to the method and controller so these issues are easier to identify when someone looks into them